packages/adaptive_scaffold/example/lib/adaptive_scaffold_demo.dart

```diff
@@ -59,14 +65,15 @@ class MyHomePage extends StatelessWidget {
             ],
             body: (_) => GridView.count(crossAxisCount: 2, children: children),
```

Comment:
Observation: none of the arguments passed use the BuildContext from the build function. I'd love it if we could just use a widget here. 

cc @HansMuller